tests/process_graphs/bands_null_graph.json

```diff
@@ -1,5 +1,5 @@
 {
-   "loadco1":{
+   "1":{
```

Comment:
Were the changes in the ids of these processes (here and test_filter_temporal.json, test_resample_cube_temporal_downsample.json, test_resample_cube_temporal_upsample.json) needed for a particular reason?

Reply:
Thanks, I understand now. Would it make sense to change them all to `loadcollection` instead of just `1`?

Reply:
Changed them to `loadcollection` but this meant also changing a few other PGs

Reply:
Thanks, looks good